### ISSUE TITLE
Missing id content on page

### ISSUE THREAD

**Issue by akibrhast** [hackforla/website#1189]
### Overview
Pages should have an id content for the screen readers to jump to main content

### Action Items
- [x] Add id of content to `_default.html`
- [x] Include a html comment within 
```html
    <a id="skip-link" className="sr-only-focusable" href="#content">
    <!-- PLEASE DO NOT REMOVE, THIS A TAG EXISTS FOR ACCESIBILITY REASONS -->

      Skip to main content.
    </a>
```

### Resources/Instructions
- https://github.com/hackforla/website/blob/gh-pages/_layouts/default.html

**Comment by evancchen** [comment]
It already has an id. I'm just adding the comment.
